ResearchKit/Consent/ORKConsentLearnMoreViewController.h

```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ORKConsentLearnMoreViewController : UIViewController
 
 - (instancetype)initWithHTMLContent:(NSString *)content;
+- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
```

Comment:
We decided that the request was what the webview uses. The developer can just send the request for the content desired.

J

Sent from my iPhone 6plus

> On May 22, 2015, at 1:29 PM, jwe-apple <notifications@github.com> wrote:
> 
> In ResearchKit/Consent/ORKConsentLearnMoreViewController.h:
> 
> > @@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
> >  @interface ORKConsentLearnMoreViewController : UIViewController
> >  
> >  - (instancetype)initWithHTMLContent:(NSString *)content;
> > +- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
> Why a request and not just a URL?
> 
> —
> Reply to this email directly or view it on GitHub.
>